students/yesol/westagram/users/views.py

```diff
@@ -36,10 +37,15 @@ class MembersLoginView(View):
     def post(self, request):
         try:
             data = json.loads(request.body)
+            member = Member.objects.filter(email=data["email"])
 
-            if not Member.objects.filter(email=data["email"]).exists() or Member.objects.get(email=data["email"]).password != data["password"] :
-                return JsonResponse({"results":"INVALID_USER"}, status=401)
-                
-            return JsonResponse({"results":"SUCCESS"}, status=201)
+            if not member.exists():
+                return JsonResponse({"results":"EMAIL_ERROR"}, status=401)
+
+            if bcrypt.checkpw(data["password"].encode("utf-8"), member[0].password.encode("utf-8")):
+                token  = jwt.encode({"user_id":member[0].id}, SECRET_KEY, ALGORITHM)
```

Comment:
```suggestion
            if not Member.objects.filter(email=data["email"]).exists():
                return JsonResponse({"results":"INVALID_USER"}, status=401)
            
            member = Member.objects.get(email=data["email"])      
        
            if bcrypt.checkpw(data["password"].encode("utf-8"), member.password.encode("utf-8")):
                token  = jwt.encode({"user_id":member.id}, SECRET_KEY, ALGORITHM)
```
- 특정 데이터 1개를 가져올 때는 filter을 통해 qeuery_set을 가져오는 것보다 get을 통해서 가져와 주세요.
- filter로 가져온 데이터일 경우 보통 복수 명사로 된 변수에 담습니다.
- 보안상 이메일이 틀렸는지, 비밀번호가 틀렸는지 모르는 것이 좋습니다. 따라서 에러 메시지를 INVALIDE_USER로 수정해 주세요.